notebooks/Existing-model-validation-example.ipynb

```diff
@@ -137,22 +137,12 @@
      "slide_type": "-"
     }
    },
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "<Graph identifier=6722b016-24df-40d2-84c9-85922fb664a6 (<class 'rdflib.graph.Graph'>)>"
-      ]
-     },
-     "execution_count": 4,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
     "BLDG = Namespace(\"http://example.org/building/\")\n",
     "medium_office_model = Model.create(BLDG)\n",
-    "medium_office_model.graph.parse(os.path.join('mediumOffice-validation', 'mediumOffice_brick_compiled.ttl'), format=\"ttl\")"
+    "medium_office_model.graph.parse(os.path.join('mediumOffice-validation', 'mediumOffice_brick_compiled.ttl'), format=\"ttl\")\n",
```

Comment:
are you sure the manifest is being properly added here? That's all I can think of for what might have changed

Reply:
i think this is right? It passes locally...

Reply:
Oh, so all these tests pass fine locally, but fail on GH actions?

Reply:
Looks like I'm getting the same error on my own PR. I'll check it out

Reply:
It looks to have resolved itself and I can't reproduce now...I think this was some caching issue internal to GitHub